app/clubs/services.py

```diff
@@ -67,10 +73,39 @@ def get_member_attendance(self, user: User):
         member = self._get_user_membership(user)
         return EventAttendance.objects.filter(member=member)
 
-    def create_event(self):
-        """Create new club event."""
-        pass
+    def create_event(self, name: str, **kwargs):
+        event_start = 2024-11-2
+        event_end = 2024-11-4
 
-    def create_recurring_event(self):
+        """Create new club event."""
+        if event_start >= event_end:
+            raise exceptions.BadRequest("The event start time must be before the end time.")
+
+        # Crear el evento
+        event = Event.objects.create(
+            club=self.club,
+            name=name,
+            event_start=event_start,
+            event_end=event_end,
+            **kwargs
+        )
+        return event
+
+    def create_recurring_event(self, name, club, start_date, end_date, day, event_start_time, event_end_time, location, description=None):
         """Create new recurring club event."""
-        pass
+        recurring_event = RecurringEvent.objects.create(
+            name=name,
+            club=club,
+            start_date=start_date,
+            end_date=end_date,
+            day=day,
+            event_start_time=event_start_time,
+            event_end_time=event_end_time,
+            location=location,
+            description=description,
+        )
+        
+        # Sincronizar los eventos individuales según el rango de fechas
```

Comment:
Try to keep comments in English since it's the lingua franca between the other devs working on the project